system.properties

```diff
@@ -1 +1,2 @@
-java.runtime.version=17
\ No newline at end of file
+java.runtime.version=17
+springdoc.swagger-ui.enabled=true
```

Comment:
Probably should not commit this because this is env specific and we should probably add this to `.gitignore`. It would still be helpful to include a system.properties.sample file though to show the file structure.